### PR TITLE
Add util class for decibel calculation

### PR DESCRIPTION
**Pull request by tong**

### CODE REVIEW

File: Sources/aura/utils/Decibel.hx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+	A change in power by a factor of 10 corresponds to a 10 dB change in level.
+	At the half power point an audio circuit or an antenna exhibits an attenuation of approximately 3 dB.
+	A change in amplitude by a factor of 10 results in a change in power by a factor of 100, which corresponds to a 20 dB change in level.
+	A change in amplitude ratio by a factor of 2 (equivalently factor of 4 in power change) approximately corresponds to a 6 dB change in level.
+**/
+class Decibel {
+
+	public static inline function toDecibel( volume : Float ) : Float {
+		return 20 * Math.log10( volume );
```
Comment (MoritzBrueckner) [inline Sources/aura/utils/Decibel.hx:17]: `Math.log10()` doesn't seem to exist, I think `Math.log(volume) / Math.log(10)` is required here?

File: Sources/aura/utils/Decibel.hx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package aura.utils;
+
+/**
+    The decibel (dB) is a logarithmic unit used to express the ratio of two values of a physical quantity.
+    One of these values is often a standard reference value, in which case the decibel is used to express the level[a] of the other value relative to this reference.
```
Comment (MoritzBrueckner) [inline Sources/aura/utils/Decibel.hx:5]: What does the [a] in level[a] refer to?

File: Sources/aura/utils/Decibel.hx (hunk 1 above)
Comment (tong) [reply Sources/aura/utils/Decibel.hx]: Added MathUtils.log10

File: Sources/aura/utils/Decibel.hx (hunk 2 above)
Comment (tong) [reply Sources/aura/utils/Decibel.hx]: Good question.
One has to find the answer in the wikipedia article history :|

**Comment by MoritzBrueckner** [comment]
Thanks a lot!

**Pull request merged**